### PR TITLE
Fix DiagnosesSCT ValueSet definition

### PR DESCRIPTION
**Pull request by ApricotLace**
# Contributor Pull Request
Fix DiagnosesSCT ValueSet definition

## Description
Fix the DiagnosesSCT ValueSet definition: the initial include statement has been split into three separate include statements.

## Motivation and Context
It appears there is a bug due to all the filters sitting beneath one include statement. Multiple filters in the same include statement assume a logical AND evaluation, and obviously, there's no concept that contains all three of these concepts in their ancestor chain.

## How has this been tested?
Initial ValueSet definition expansion returns nothing:
https://r4.ontoserver.csiro.au/fhir/ValueSet/32367c12-3e08-46a4-bfc9-c84b9849efb9/$expand

Updated ValueSet definition expansion returns concepts:
https://r4.ontoserver.csiro.au/fhir/ValueSet/de18023b-0d33-4ad1-a5d1-dae1f02d3dc7/$expand

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this IG / specification.
- [ ] My change requires a change to the documentation or narrative (intend) of the IG.
- [ ] I have already updated the documentation / narrative (intend) accordingly.

### CODE REVIEW

**Comment by ApricotLace** [comment]
It looks like this fix should also be backported to the Stufe-1 and Stufe-2 tracks.

File: Resources/input/fsh/valueSets.fsh [hunk 1]
```diff
@@ -3,9 +3,11 @@ Id: DiagnosesSCT
 Description: "Enthaelt alle SNOMED Clinical finding, Event und Situation with explicit context codes"
 * insert Meta
 * include codes from system SNOMED_CT
-    where concept is-a #404684003 and
-    concept is-a #272379006 and
-    concept is-a #243796009
+    where concept is-a #404684003
```
Comment (f-peverali) [inline Resources/input/fsh/valueSets.fsh:6]: praise: @ApricotLace , thanks for the correction

**Review by f-peverali — APPROVED** [review]

**Comment by alexzautke** [comment]
@f-peverali Could you please create a new branch and cherry-pick the changes? We can't run the pipeline from outside repos.

**Comment by f-peverali** [comment]
In order to execute the gh actions, i created another PR: https://github.com/gematik/spec-ISiK-Basismodul/pull/400 ... looks fine

**Pull request merged**